### PR TITLE
 Gmvault: gmail backup gaubert.gmvault version 1.5-beta

### PR DESCRIPTION
**Pull request by richardsondev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16161)

### CODE REVIEW

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml:6]:
```suggestion
- Architecture: neutral
```

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml:4]:
```suggestion
PackageVersion: 1.5-beta
```

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.5-beta
```

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
+PackageLocale: en-US
+Publisher: Guillaume Aubert (gaubert)
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:6]:
```suggestion
Publisher: Guillaume Aubert
```

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
+PackageLocale: en-US
+Publisher: Guillaume Aubert (gaubert)
+PackageName: Gmvault Google Gmail Backup
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:7]:
```suggestion
PackageName: Gmvault
```

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1
```
Comment (OfficialEsco) [inline manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.yaml:4]:
```suggestion
PackageVersion: 1.5-beta
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Weird, the version dosn't match the version it installs..
Remember to change the folder name.

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml (hunk 2 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml:4]: Thanks for your help

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml (hunk 1 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml:6]: Got it!

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml (hunk 3 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:4]: Updated here too

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml (hunk 4 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:6]: I will keep this in mind in the future

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml (hunk 5 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.locale.en-US.yaml:7]: Updated! 👍

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.yaml (hunk 6 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.yaml:4]: Got it here too and the folder

File: manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml (hunk 2 above)
Comment (richardsondev) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml]: I had noticed this as well but decided to take the version on the website of the download. I will keep this in mind for future PRs.

Comment (OfficialEsco) [reply manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml]: Yeeeah WinGet looks at the Version which shows up in the Control Panel, so for now use that.

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**